### PR TITLE
Add accessibility meeting notes for end of 2022 and beginning of 2023

### LINKED ISSUE #132: capturing the accessibility meeting minutes in the repo
we haven't captured meeting notes in the repo all year. is there anyone that has the time and focus to go through the back log of notes that have not made it into the repo? 

we stopped adding notes on 9/2022 

the notes are still running the same https://hackmd.io/WnaWXboXSiGoqWvev_fAvA

#119 has complicated the process a bit i feel.

### PR DESCRIPTION
**Pull request by isabela-pf**
I'm adding accessibility meeting notes! There's a lot of notes, so this PR looks big. If it should be broken up instead, let me know.

This PR adds:
- Fixes to the 2022 meeting notes index to make the existing notes appear in the docs (some were not appearing)
- The notes from the end of 2022 to the existing directory and index
- The notes for 2023 up to August 2023 in a new directory

Thanks in advance for the review.

Fixes #132 

----
:books: Documentation preview :books:: https://jupyter-accessibility--137.org.readthedocs.build/en/137/

### CODE REVIEW

**Review by afshin — APPROVED** [review]
Thank you @isabela-pf!

**Comment by isabela-pf** [comment]
Thanks for the review! It made me realize I was missing some links to complete the loop.

**Comment by tonyfast** [comment]
thanks @isabela-pf 

closes #132

**Comment by isabela-pf** [comment]
I mentioned on this week's accessibility meeting that I plan on self-merging this PR if there's no comment or review otherwise in the next week. As far as I can tell this PR doesn't break anything so it's very iteratable if we find an error later. Thank you!

File: docs/community/meeting-minutes/jupyterlab-accessibility-meetings/2022/index.md [hunk 1]
```diff
@@ -6,14 +6,23 @@
 
 ````{include} 2022-01-26.md
```
Comment (gabalafou) [inline docs/community/meeting-minutes/jupyterlab-accessibility-meetings/2022/index.md:7]: Why does this one still have four back ticks? Shouldn't all the `{include}`s be triple-back-ticked?

File: docs/community/meeting-minutes/jupyterlab-accessibility-meetings/all-minutes.md [hunk 2]
```diff
@@ -2113,3 +2113,915 @@ Jupyter governance and jupyter/accessibility. A discussion around what's happeni
...
+        * Put accessibility concerns earlier (a11y in the planning phase)
+            * and consider at every phase of development
+        * We recognize that this is not simple to scope, but neither is security
+        * Funding for events to teach developers about accessibility 
+        * What happens if Jupyter fails to close accessibility gaps and ed institutions are forced to not use it?
+* Gabriel's update:
+    * Took a PR out of draft mode that [removes tab traps from the Notebook widget](https://github.com/jupyterlab/jupyterlab/pull/14115)
+        * On the PR there is a [discussion about how to style focus indicators on the code cell](https://github.com/jupyterlab/jupyterlab/pull/14115#pullrequestreview-1409130132)
+        * [Tony] https://www.sarasoueidan.com/blog/focus-indicators/
+## May 17, 2023
```
Comment (gabalafou) [inline docs/community/meeting-minutes/jupyterlab-accessibility-meetings/all-minutes.md:2668]: newline?
```suggestion 

## May 17, 2023
```

**Review by gabalafou — CHANGES_REQUESTED** [review]
I performed the following checks:

- [ ] Dates in the 2022 and 2023 indexes match the files shown in GitHub left sidebar
- [ ] Date in heading and date of filename all match
- [ ] All dates accounted for and in order in headings add to all-minutes.md
- [ ] Scanned the beginning of each agenda looking to see if I spotted any accidental repeats

Marking request changes because I have a couple of questions/suggestions.

**Comment by tonyfast** [comment]
thanks for this huge amount of work isa!

**Pull request merged**